apps/blockchain/lib/blockchain/account.ex

```diff
@@ -6,6 +6,7 @@ defmodule Blockchain.Account do
 
   alias ExthCrypto.Hash.Keccak
   alias MerklePatriciaTree.Trie
```

Comment:
Since we're moving to use `TrieStorage` as the interface for interacting with the Trie, would it make sense to remove this and force account to interact with the Trie via `TrieStorage`?

Reply:
this alias is used for types from Trie module